spec/controllers/api/teams_controller_spec.rb

```diff
@@ -381,6 +414,47 @@
         end
       end
     end
+
+    context 'Receive transferred encryptables' do
+      it 'Receive method gets called to encrypt transferred encryptable' do
+        login_as(:bob)
+
+        prepare_transferred_encryptable
+
+        expect_any_instance_of(EncryptableTransfer)
+          .to receive(:receive)
+          .exactly(1).times
+          .and_return(nil)
+
+        default_folder = Folder.new(name: 'default', team: personal_team_bob)
+
+        personal_team_bob.folders = [default_folder, bob.inbox_folder]
+
+        get :index, params: { team_id: personal_team_bob.id }, xhr: true
+      end
+
+      it 'Receive method gets not called if encryptables are not transferred' do
```

Comment:
try not to describe test cases in a technical way. 

what does it or does it not in a user perspective?